tools/code_timer.py

```diff
@@ -0,0 +1,34 @@
+# code_timer.py
```

Comment:
I wasn't quite sure what a "code timer" is. Perhaps just call it benchmark?

Reply:
fixed in f05521cc0cc08cf2d03c2459535dc22ae8f0f5db